XenAdminTests/MiscTests/WebBrowser2Tests.cs

```diff
@@ -157,10 +141,20 @@ public void TestMultipleUrisWithNoneValid()
                 navError = true;
             };
 
-            var uris = new[] { uri1, uri2 };
+            //Use non-existing URLs with existing rather than completely fictional
+            //domains, otherwise the NavigationError even is not fired
+
+            var uris = new[]
+            {
+                new Uri("http://127.0.0.1/blah"),
+                new Uri("http://127.0.0.1/blahblah")
+            };
+
             foreach (var uri in uris)
             {
-                var curUri = uri;
+                curUri = uri;
+
+                navigated = navigated = navError = false;

```

Comment:
was this supposed to include `navigating` (instead of one of the `navidated`) ?

Reply:
Oops, yes 😳